src/main/webapp/user_profile.html

```diff
@@ -6,40 +6,73 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getUserInfo()">
     <div id="navigation">
       <a href="main_hub.html" id="hub">Main Hub</a>
       <a href="user_profile.html" id="user">User Profile</a>
       <a href="project.html" id="project">Project Page</a>
       <a href="task.html" id="task">Task Page</a>
     </div>
     <div id="content">
-      <h1>Welcome, UserName</h1>
+      <h1 id="user-name-container">Default User Name</h1>
       <h2>User Stats:</h2>
-      <div id="user-stats">
-        <div>Name: UserName LastName</div>
-        <div>Year: Junior</div>
-        <div>Major: Biology</div>
-        <div>Total Completed Tasks: 7</div>
-        <button type="button" onclick="goToTask()">See Tasks</button>
+      <div class="user-stats">
+        <div id="year-container">Year: Default Year</div>
+        <div id="major-container">Major: Default Major</div>
+        <div id="num-complete-container">Total Completed Tasks: default</div>
```

Comment:
I styled the container ids as "page-type-container", for example "project-admins-container". You don't have to do the same thing, but you can change the ids to something like "user-year-container" for consistency. (And this helps avoiding confusion just in case someone would want to make a "year-container" on another page.)